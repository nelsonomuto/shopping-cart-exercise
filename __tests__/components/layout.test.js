import { shallow } from "enzyme"
import React from "react"
import Layout from "../../src/components/layout"

describe("<Layout />", () => {
  it("matches snapshot", () => {
    const component = shallow(<Layout />)
    expect(component).toMatchSnapshot()
  })
  it("renders children", () => {
    const Child = () => <span>Test</span>
    const component = shallow(
      <Layout>
        <Child />
      </Layout>
    )
    expect(component.contains(<Child />)).toBeTrue
  })
})
