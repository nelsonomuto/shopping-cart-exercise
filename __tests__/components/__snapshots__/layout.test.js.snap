// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> matches snapshot 1`] = `
<Fragment>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <link
      crossorigin="anonymous"
      href="https://cdn.jsdelivr.net/npm/foundation-sites@6.5.3/dist/css/foundation.min.css"
      integrity="sha256-xpOKVlYXzQ3P03j397+jWFZLMBXLES3IiryeClgU5og= sha384-gP4DhqyoT9b1vaikoHi9XQ8If7UNLO73JFOOlQV1RATrA7D0O7TjJZifac6NwPps sha512-AKwIib1E+xDeXe0tCgbc9uSvPwVYl6Awj7xl0FoaPFostZHOuDQ1abnDNCYtxL/HWEnVOMrFyf91TDgLPi9pNg=="
      rel="stylesheet"
    />
  </HelmetWrapper>
  <CartStoreProvider>
    <Header
      siteTitle="RTG"
    />
    <div
      style={
        Object {
          "margin": "0 auto",
          "maxWidth": 960,
          "padding": "0px 1.0875rem 1.45rem",
          "paddingTop": 0,
        }
      }
    >
      <main />
      <footer />
    </div>
    <CartSidebar />
  </CartStoreProvider>
</Fragment>
`;
